tests/server/wildfly/src/test/java/org/hawkular/apm/tests/wildfly/TraceServiceRESTTest.java

```diff
@@ -164,12 +165,8 @@ public void testStoreAndRetrieveComplexTraceById() {
         p2_2.addInteractionId("id2_2");
         c2.getNodes().add(p2_2);
 
-        List<Trace> traces = new ArrayList<Trace>();
-        traces.add(trace1);
-        traces.add(trace2);
-
         try {
-            publisher.publish(null, traces);
+            publisher.publish(null, Arrays.asList(trace1, trace2));
```

Comment:
Does it make sense to add an alternate `publish` for single traces? It seems it's far more common now than before?